docs/openapi/components/schemas/common/MonetaryAmount.yml

```diff
@@ -0,0 +1,47 @@
+$linkedData:
+  term: MonetaryAmount
+  '@id': https://schema.org/MonetaryAmount
+title: Monetary Amount
+description: A monetary value or range. This type can be used to describe an amount of money such as $50 USD, or a range as in describing a bank account being suitable for a balance between £1,000 and £1,000,000 GBP, or the value of a salary, etc. It is recommended to use PriceSpecification Types to describe the price of an Offer, Invoice, etc.
```

Comment:
Note that `$50 USD` and `£1,000,000 GBP` have redundant units. The general recommendation is to use the three-character currency abbreviation (which is unique to one nation/currency) and no symbol (as these are often overloaded, particularly `$`) as in `USD 50` and `GBP 1,000,000`